src/tests/integration-test

```diff
@@ -333,13 +333,13 @@ class UDisksTestCase(unittest.TestCase):
         block.call_format_sync(fs_type, options, None)
 
     @classmethod
-    def retry_busy(cls, fn, *args):
+    def retry_busy(cls, fn, *function_args):
```

Comment:
Well, that's what the local scope is for, right? I don't see as a problem, really. But if you want to rename one of those, rename the variable in ``__main__`` as ``*args`` is a common name, just like ``self``.